tailwind.config.js

```diff
@@ -12,21 +12,37 @@ module.exports = {
         5.5: '1.375rem',
         7.5: '1.875rem',
       },
+      fontFamily: {
+        mercury: ['"PP Object Sans", sans-serif'],
+        'mercury-alt': ['"PP Monument Extended", sans-serif'],
+      },
       fontSize: {
         '2xs': ['0.625rem', '0.9375rem'],
+        'xs': '.625rem', // 10px
+        'sm': '.75rem', // 12px
+        'base': '.875rem', // 14px
+        '2base': '1rem', // 16px
+        '3base': '1.125rem', // 18px
+        'lg': '1.5rem', // 24px
+        'xl': '2rem', // 32px
+        '2xl': '2.5rem', // 40px
+        '3xl': '2.625rem', // 42px
+        '4xl': '3rem', // 48px
+        '5xl': '3.375rem', // 54px
+        '6xl': '3.75rem', // 60px
+        '7xl': '4.25rem', // 68px
```

Comment:
I think we should minimize the amount of overrides we make on the default settings, and some the font sizes here look like the default values but "shifted" 1 place. Also text-sm, text-base are used in buttons already an these overrides impact them.